src/client/Microsoft.Identity.Client/MsalServiceException.cs

```diff
@@ -224,7 +224,7 @@ public HttpResponseHeaders Headers
         /// <summary>
         /// As per discussion with Evo, AAD 
         /// </summary>
-        private void UpdateIsRetryable()
+        protected virtual void UpdateIsRetryable()
         {
             IsRetryable =
                 (StatusCode >= 500 && StatusCode < 600) ||
```

Comment:
501 and 505 will also be in this condition, which should not be retried

Reply:
This is an existing retry logic for ests requests. Changing this logic is a breaking change.

Reply:
Just FYI, ESTS currently throws 500 and 503.